src/applications/claims-status/components/claim-files-tab/AdditionalEvidencePage.jsx

```diff
@@ -120,39 +122,27 @@ class AdditionalEvidencePage extends React.Component {
           {isOpen ? (
             <>
               {this.props.filesNeeded.map(item => (
-                <FilesNeeded
-                  key={item.id}
-                  id={this.props.claim.id}
-                  item={item}
-                />
+                <FilesNeeded key={item.id} id={claim.id} item={item} />
               ))}
               {this.props.filesOptional.map(item => (
-                <FilesOptional
-                  key={item.id}
-                  id={this.props.claim.id}
-                  item={item}
-                />
+                <FilesOptional key={item.id} id={claim.id} item={item} />
               ))}
               <AddFilesForm
                 field={this.props.uploadField}
                 progress={this.props.progress}
                 uploading={this.props.uploading}
                 files={this.props.files}
-                backUrl={this.props.lastPage || filesPath}
+                backUrl={lastPage || filesPath}
                 onSubmit={() => {
                   // START lighthouse_migration
                   if (this.props.documentsUseLighthouse) {
                     this.props.submitFilesLighthouse(
-                      this.props.claim.id,
+                      claim.id,
                       null,
                       this.props.files,
```

Comment:
you could also pull out the  files from this.props to clean that up as well since its used more then once

Reply:
Yeah that's a good call, will probably get to that one in the future